templates/article/article.js

```diff
@@ -3,24 +3,25 @@ import {
 } from '../../scripts/lib-franklin.js';
 
 export default async function decorate(doc) {
+  // remove title and image from doc
+  doc.querySelector('main .section h1').remove();
+  doc.querySelector('main .section img').remove();
```

Comment:
This breaks the video hero variant. I think this code needs to moved down into the `if (!videoHero) {`